src/era_5g_relay_network_application/era_5g_relay_network_application/utils.py

```diff
@@ -5,17 +5,25 @@
 
 from typing import List, Tuple, Optional, Dict
 
+from era_5g_interface.channels import ChannelType
+from rosbridge_library.internal import ros_loader
+from sensor_msgs.msg import Image
+
 class Compressions(str, Enum):
     NONE = 'none'
     LZ4 = 'lz4'
     DRACO = 'draco'
+    JPEG = 'jpeg'
+    H264 = 'h264'
 
     @classmethod
     def _missing_(cls, value):
         if value is None:
             return cls('none')
         return super()._missing_(value)
     
+IMAGE_CHANNEL_TYPES = [ChannelType.JPEG, ChannelType.H264]
```

Comment:
Use some immutable type (tuple, frozenset)?

Reply:
Changed to tuple